src/core/send.c

```diff
@@ -987,6 +987,8 @@ QuicSendFlush(
     QuicConnRemoveOutFlowBlockedReason(
         Connection, QUIC_FLOW_BLOCKED_SCHEDULING | QUIC_FLOW_BLOCKED_PACING);
 
+    QuicMtuDiscoveryCheckSearchCompleteReset(&Connection->MtuDiscovery);
```

Comment:
Might be better to put this a little farther down, after all the return checks.

Reply:
This might actually add flags. So this has to go before the line right below it, which checks the send flags.